package.json

```diff
@@ -12,12 +12,13 @@
     "test:watch": "npm run test:local -- --watch",
     "test:coverage": "npm run test:local -- --coverage",
     "test:update": "npm run test:local -- --updateSnapshot",
-    "start": "webpack-dev-server --env.NODE_ENV=dev --progress --colors",
+    "start:dev": "webpack-dev-server --config webpack/webpack.config.js --env.NODE_ENV=dev --progress --colors",
+    "start:prev": "webpack-dev-server --config webpack/webpack.config.js --port=9000 --env.NODE_ENV=prev --progress --colors",
     "lint:es": "eslint '*/**/*.js'",
     "lint:ts": "tslint -c tslint.json 'src/**/*.{ts,tsx}'",
-    "build:dev": "webpack --env.NODE_ENV=dev --progress --profile --colors",
-    "build:prod": "webpack --env.NODE_ENV=prod --progress --profile --colors",
-    "build:both": "npm run build:dev && npm run build:prod",
+    "build:dev": "webpack --config webpack/webpack.config.js --env.NODE_ENV=dev",
```

Comment:
Webpack-specific files are now grouped in the `/webpack` directory. The `--config` flag allows us to make Webpack aware of configs that don't live in the root directory.